cedar-policy/src/api.rs

```diff
@@ -3904,6 +3904,127 @@ mod schema_based_parsing_tests {
     use cool_asserts::assert_matches;
     use serde_json::json;
 
+    #[test]
+    fn long_in_bounds() {
+        let schema = Schema::from_json_value(json!(
+        {"": {
+            "entityTypes": {
+                "Employee": {
+                    "memberOfTypes": [],
+                    "shape": {
+                        "type": "Record",
+                        "attributes": {
+                            "num": { "type": "Long", "min": 3, "max": 4 }
```

Comment:
Not related to this PR, but what did we do regarding making `min` and `max` optional but requiring both if one is given?